backend/src/main/java/gov/cdc/usds/simplereport/db/model/AuditedEntity.java

```diff
@@ -33,11 +34,13 @@ public abstract class AuditedEntity extends IdentifiedEntity {
   @Immutable // not sure this is needed. Not sure it works if it is. :-(
   @ManyToOne(optional = false, fetch = FetchType.LAZY)
   @JoinColumn(name = "created_by", updatable = false)
+  @JsonIgnore
   private ApiUser createdBy;
 
   @LastModifiedBy
   @ManyToOne(optional = false, fetch = FetchType.LAZY)
   @JoinColumn(name = "updated_by")
+  @JsonIgnore
```

Comment:
Since I added `getCreatedBy() `and `getUpdatedBy()`, these fields will be included into the jsonb that saved in the db unless i specifically ignore it. 
Normally this wouldn't be a big deal, but since we wanna maintain a **forward compatibility** on main with the data that created in this branch (in case of a revert), I chose to remove it for the json

Reply:
Would we want to change this and remove the `@JsonIgnore` in the future?

Reply:
No, we don't currently save it in the jsonb, and there is no reason to start to. To be clear, the `createdBy`/`updatedBy` fields are saved in the DB for each entity so there is no need to duplicate it. We are duplicating some data in the TestEvents as an effort to make them a complete final record that doesn't rely on any other tables.

Reply:
thanks, that makes sense now!